### PR TITLE
[enhance](mtmv)not allow modify data of MTMV

### PR DESCRIPTION
**Pull request by zddr**
Prohibited scope：
- Transaction related operations,such as:insert,update,delete
- insert overwrite
- truncate table

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run feut

**Review by morrySnow — APPROVED** [review]

File: regression-test/suites/mtmv_p0/test_modify_data_mtmv.groovy [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        DISTRIBUTED BY RANDOM BUCKETS 2
+        PROPERTIES (
+        'replication_num' = '1'
+        )
+        AS
+        SELECT * from ${tableName};
+        """
+
+    sql """
+        insert into ${tableName} values(1,1),(2,2),(3,3);
```
Comment (morrySnow) [inline regression-test/suites/mtmv_p0/test_modify_data_mtmv.groovy:50]: should also test stream load, routine load, broker load, update, delete and truncate

Comment (zddr) [reply regression-test/suites/mtmv_p0/test_modify_data_mtmv.groovy:50]: i will do it in other pr

**Review by morningman — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**